Gulpfile.js

```diff
@@ -55,8 +67,57 @@ function build() {
     .pipe(gulp.dest(base));
 }
 
+async function buildRollup() {
+  const base = path.join(__dirname, "packages");
+
+  return Promise.all(
+    packages.map(async name => {
+      const dir = `${base}/${name}`;
+      const pkg = require(`${dir}/package.json`);
+      const external = specifier => {
+        if (specifier.includes("/core-js-compat/")) return true;
+        if (specifier.includes("/data/polyfills")) return true;
+        if (specifier.startsWith("/")) return false;
+        const name = /^(?:@[^/]+\/[^/]+|[^/]+)/.exec(specifier)?.[0];
+        if (name === "@babel/core") return true;
+        return Object.hasOwnProperty.call(pkg.dependencies || {}, name);
+      };
+
+      const bundle = await rollup({
+        input: `${dir}/src/index.js`,
+        external,
+        plugins: [
+          rollupJson(),
+          rollupBabel({
+            envName: "esm",
+            babelrc: false,
+            babelHelpers: "bundled",
+            extends: "./babel.config.js",
+            extensions: [".ts", ".js", ".mjs", ".cjs"],
+          }),
+          rollupNodeResolve({
+            extensions: [".ts", ".js", ".mjs", ".cjs", ".json"],
+            browser: true,
```

Comment:
Q: Why do we need to resolve against browser fields?

Reply:
We now provide two bundles for `@babel/helper-define-polyfill-provider`: one for Node.js (which accesses the filesystem) and one for the browser (which doesn't).